sdk/monitor/Azure.Monitor.Ingestion/api/Azure.Monitor.Ingestion.net461.cs

```diff
@@ -7,9 +7,9 @@ public partial class LogsIngestionClient
         public LogsIngestionClient(System.Uri endpoint, Azure.Core.TokenCredential credential, Azure.Monitor.Ingestion.LogsIngestionClientOptions options) { }
         public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
         public virtual Azure.Response Upload(string ruleId, string streamName, Azure.Core.RequestContent content, string contentEncoding = null, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Response Upload(string ruleId, string streamName, System.Collections.Generic.IEnumerable<object> logs, Azure.Monitor.Ingestion.UploadLogsOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Curious why do we have different APIVersions for ney461 and .netstandard2.0 Are they different in anyway? Or is this just because the SDK multi-targets?

Reply:
We pull in shared source at https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/core/Azure.Core/src/Shared which multitargets at https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/Shared/GZipUtf8JsonRequestContent.cs#L44 and we end up with separate API compat listings.

We should probably file a work item to have the CI automatically diff the files under the `/api/*` folders to ensure they're identical so we never diverge the public API across runtimes.

Reply:
Created: https://github.com/Azure/azure-sdk-for-net/issues/33830